content/docs/overview.md

```diff
@@ -696,8 +696,14 @@ create_window :: proc(title: string, x, y: int, width, height: int, monitor: ^Mo
 window, err := create_window(title="Hellope Title", monitor=nil, width=854, height=480, x=0, y=0)
 ```
 
-**Note:** Currently, mixing named and non-named arguments is not allowed. This is subject to change if it is deemed necessary.
+Since the release of `dev-2023-07` mixing named and non named arguments is allowed. This is often useful when a procedure has a lot of arguments *or* you want to customize [default values](#default-values). 
```

Comment:
I'd call them positional arguments, so "As of the `dev-2023-07` release, mixing named and positional arguments is allowed".